scripts/install.sh

```diff
@@ -19,7 +19,7 @@ fi
 
 if [ ! -z $2 ]; then UNIQUE=$2; fi
 if [ -z $UNIQUE ]; then
```

Comment:
Was looking for something that works in many scenarios and I can't seem to find one.  So what about this as an option.

```bash
if [ ! -z $2 ]; then UNIQUE=$2; fi
if [ -z $UNIQUE ]; then
  if [ -x "$(command -v shuff)" ]; then
    UNIQUE=$(shuff -i 100-999 -n 1)
  elif [ -x "$(command -v jot)" ]; then
    UNIQUE=$(jot -r 1 100 999)
  else
    UNIQUE=$(head -c 10000 /dev/urandom | tr -dc '0-9' | fold -w 256 | head -n 1 | sed -e 's/^0*//' | head --bytes 3)
  fi
  echo "export UNIQUE=${UNIQUE}" >> .envrc
fi
```